simulator-archetypes/archetype-jms/src/main/resources/archetype-resources/src/main/java/scenario/DefaultScenario.java

```diff
@@ -16,8 +16,6 @@
 
 package ${package}.scenario;
 
-import com.consol.citrus.simulator.scenario.*;
-
```

Comment:
I think we should refactor the import to use `org.citrusframework` instead of deleting it

Reply:
hm.. it is not required. for the sake of example?